src/main/java/de/qabel/desktop/config/ClientConfiguration.java

```diff
@@ -34,4 +35,7 @@
 	 * @see java.util.Observable#deleteObserver(Observer)
 	 */
 	void deleteObserver(Observer o);
+
+	Date getLastUpdate();
```

Comment:
This name is very misleading.
Obvious assumption would be "lastUpdate of the ClientConfiguration", which is false.
Next assumption could be "lastUpdate of the Identity / BoxAccount" .. also false.

A name like "lastDropPoll" or "lastDropUpdate" would be more precise.
But ... doesn't it need to be tied to the identity?
If I have two identities and identity 1 is selected, then lastUpdate will be about `now`.
When someone sends messages to identity 2 simultaniously, but only the dropUrls of identity 1 are checked, the messages of identity 2 will be lost / never fetched.